packages/embed/src/utils/set-element-size.ts

```diff
@@ -0,0 +1,11 @@
+export const setElementSize = (element: HTMLElement, { width, height }: { width?: number; height?: number }) => {
```

Comment:
Please do not define interfaces inline like this, it is harder to read.

```suggestion
interface ElementSize {
  width?: number
  height?: number
}

export const setElementSize = (element: HTMLElement, { width, height }: ElementSize) => {
```

Or you can pass each one as separate variable.

Reply:
Since we have `setElementSize` function extracted, lets use it in all places where we set element sizes -> popover and slider too.

Reply:
already did, thing is slider has a slightly different implementation also popover

Reply:
Slider is doing this:

```
wrapper.style.width = `${width}px`
return wrapper
```

You could do 

```
return setElementSize(wrapper, { width })
```

Reply:
Sorry I meant popup not popover. Something like this:

```
   if (isDefined(width) && isDefined(height)) {
-    wrapper.style.width = `${width}px`
-    wrapper.style.height = `${height}px`
+    return setElementSize(wrapper, {width, height)
   }
```